changelog/1.16.0.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "1.16.0"
+description: "Released on 02/22/2021"
+---
+
+### Breaking Changes ❗
+
+- The deprecated `Services` feature is now **removed**. All resources created
+  for Services are preserved for data recovery purposes (volumes,
+  database tables, etc.), but are not accessible through the Coder platform.
+  Please reference
+  [Docker in Environments](../environments/cvms) for
+  a guide on the new workflow for running containers with your Environment.
+- The built-in Internal Extension Marketplace is now **removed**. All resources
+  created for this feature are preserved for data recover purposes, but are not
+  accessible through the Coder platform. For air-gapped deployments, the `Custom`
+  Extension Marketplace configuration enables the use of
+  [Open VSX](https://github.com/eclipse/openvsx) as
+  an alternative. For more details, please view our [migration guide](TODO).
+- The administrator configuration option "Privileged Environments" is now
+  removed. All non-CVM environments are launched with `privileged: false`.
+- The [Container-based Virtual Machine](../admin/environment-management/cvms)
+  environment option is now considered generally available. The
+  admin-level configuration is still required, but the `alpha`
+  warning has been removed.
+- Environment SSH routing has new behavior. Please reference
+[this guide](../admin/environment-management/ssh-access.md).
+
+### Features ✨
+
+- Public Rest API <apidocs.coder.com>
+- Support for remote port forwarding over SSH
```

Comment:
We should link to a guide or provide an example here.

Reply:
#69 could be referenced too

Reply:
It is to be added later as I understand things